docs/content/menu/index.md

```diff
@@ -43,6 +43,7 @@ headless: true
     - [Test minimized]({{< ref "/test/TestMinimized.md" >}})
     - [Troubleshooting]({{< ref "/test/TroubleshootingUI.md" >}})
     - [Кастомный запуск]({{< ref "/test/DynamicConfig.md" >}})
+    - [Кастомный клик]({{< ref "/test/InProcessClick.md" >}})
```

Comment:
no one is interested in implementation details on this level in menu.
Let's wrap it in more generic structure:

```
Testing
  ...
  - Custom running
  - Actions
```

Reply:
Generally, it's better to start from common pages and split them into multiple when they come too big.
I'm thinking about other actions - swipe, text, ... For beginning, one page for all actions looks enough.

Reply:
I agree that nobody except us need those details. But i don't understand where do you want to see that page.

Reply:
I suggest to copy structure of documentation from Espresso:
https://developer.android.com/training/testing/espresso/basics

When it becomes bigger, we will split pages.